webpack.config.js

```diff
@@ -86,6 +86,13 @@ module.exports = {
   },
 
   plugins: [
+    new MiniCssExtractPlugin({
+      filename: function (config) {
+        if (config.chunk.name == "pdf") return "[name].css";
+        else return "[name].[contenthash].css";
+      },
+      chunkFilename: "[id].[contenthash].css",
+    }),
```

Comment:
```suggestion
    new MiniCssExtractPlugin({
      filename: "[name].[contenthash].css",
      chunkFilename: "[id].[contenthash].css",
    }),
```
I did the exact same thing in the map PR, copied&pasted without realizing we aren't handling PDFs here 😄